Assets/Plugins/LeapMotion/Core/Shaders/InteractionVolumeWireframe.shader

```diff
@@ -0,0 +1,100 @@
+﻿Shader "LeapMotion/Geometry/Wireframe"
+{
+    Properties
+    {
+        [PowerSlider(3.0)]
+        _WireframeVal("Wireframe width", Range(0., 0.5)) = 0.2
+    }
+
+    SubShader
+    {
+        Tags { "Queue" = "Geometry" "RenderType" = "Opaque" }
+
+        Pass
+        {
+            Cull Back
+            CGPROGRAM
+            #pragma vertex vert
+            #pragma fragment frag
+            #pragma geometry geom
+
+            #include "UnityCG.cginc"
+
+            struct v2g {
+                float4 worldPos : SV_POSITION; 
+                float  cameraToSurfaceNormalAngleDegrees : PSIZE;
+            };
+
+            struct g2f {
+                float4 pos : SV_POSITION;
+                float3 bary : TEXCOORD0;
+                float4 lineColour : TEXCOORD1;
+            };
+
+            v2g vert(appdata_base v) {
+                v2g o;
+                o.worldPos = mul(unity_ObjectToWorld, v.vertex);
+
+                float3 worldNorm = UnityObjectToWorldNormal(v.normal);
+                float3 cameraForwardVector = mul((float3x3)unity_CameraToWorld, float3(0, 0, 1));
+
+                o.cameraToSurfaceNormalAngleDegrees = degrees(acos(dot(normalize(worldNorm), normalize(cameraForwardVector))));
```

Comment:
Here you can see that you're calculating the angle between the surface normal and the cameraForwardVector, which is almost right, except it means you're going to get an angle of 180 degrees when you're looking right at the normal.

The simple fix is to add a `-` in front of `worldNorm`.   This ensures that the shader works properly without any mesh-level normal flipping required.